docs/cohorts.md

```diff
@@ -8,23 +8,94 @@ hide_title: true
 
 # Cohorts
 
-The cohorts described here are intended to support multiple use cases related to COVID-19.
+The cohorts described here are intended to support use cases related to COVID-19. Searches are defined in terms of standard codes, but text searches and local code mapping may be necessary to retrieve the all members of the cohort.
 
-Each cohort will be cross-referenced to a OHDSI/ATLAS cohort.
+### Active Patient
 
-> Notice:
-This content is rough draft or placeholder quality, and is expected to change quickly as we work with C19HCC partners to align and refine definitions.
+A patient who has interacted with a health system within two years of index date and alive at the index date:
 
+  * One or more recorded encounters within two (2) years of the index date, including face-to-face (inpatient or outpatient) encounters, and telehealth encounters, AND
+  * No death recorded during the same period
+
+### COVID-19 Positive
+
+A patient diagnosed with COVID-19:
+
+  * Any clinical diagnosis code in set [Confirmed COVID-19 Infection VS](/docs/vocabulary#confirmed-covid-19-infection-vs) OR
+  * `Positive` or `detected` result from any test in set [SARS Coronavirus 2 Qualitative Laboratory Test VS](/docs/vocabulary#sars-coronavirus-2-qualitative-laboratory-test-vs) OR
+  * `Positive` or `detected` result from any test in set [SARS-Related Qualitative Laboratory Test VS](/docs/vocabulary#sars-related-qualitative-laboratory-test-vs)
+
+### COVID-19 Positive Date
+
+The earliest date of a COVID-19 diagnosis:
+
+* First date of [COVID-19 Positive](#covid-19-positive)
+
+### COVID-19 Related Hospitalization
+
+An inpatient visit associated with COVID-19:
+
+  * [COVID-19 Positive Date](#covid-19-positive-date) during hospitalization OR
+  * Hospitalization up to 14 days after [COVID-19 Positive Date](#covid-19-positive-date) AND clinical or billing diagnosis of [Respiratory Condition VS](/docs/vocabulary#respiratory-condition-vs) associated with the hospitalization
+
+### Intravesical BCG
+
+Administration of BCG vaccine into intravesical space:
+
+* Medication administration code in [BCG Vaccine Products for Intravesical Use VS](/docs/vocabulary#bcg-vaccine-products-for-intravesical-use-vs) AND
+* Medication administration route in [Intravesical Route VS](/docs/vocabulary#intravesical-route-vs)
```

Comment:
The route is precoordinated in the code.

Reply:
Agreed.

Reply:
This also means the Intravesical Route VS can be eliminated.